### ISSUE TITLE
Have an option to be able to generate just the client side of the stack

### ISSUE THREAD

**Issue by colinbull** [SAFE-Stack/SAFE-template#56]
It would be really nice to have an option in the template to just generate the client side or the server side. 

Typically this issue arises when we have an existing or external API and we are building a mashup, of several data sources.

I realize I could just use the fable template and reference the Elmish,Fulma libs etc. If that ends up being the best place to maybe do that then that is fine, but this feels the most natural place to do it. Even if it is a slight departure from SAFE being an end-to-end stack. 

What do you think?

**Comment by isaacabraham** [comment]
Yep, that's a good idea. The only concern I have personally is - as you say - that it becomes harder to know "which" template to use. However, I'd be quite happy with the SAFE template becoming the defacto home for web-enabled applications anyway. We could always start with just e.g. server side perhaps? @theimowski what do you think?

**Comment by theimowski** [comment]
Seems reasonable, but that would certainly require more effort - currently template has only a couple of switches and is already a bit complex in regards of maintenance. 

What I'm also think of (not sure whether possible to do with templating engine) is to have a "modular" templates, say a separate template for back-end, one could invoke e.g. `dotnet new SAFE.Server` and separate for front-end e.g. `dotnet new SAFE.Client` and then a "full-stack `dotnet new SAFE` which could combine the earlier two.

**Comment by isaacabraham** [comment]
I'm wondering whether it's better to simply point people towards the templates that already exist for Saturn, Giraffe and Fable etc.?

**Comment by theimowski** [comment]
I think that would be best option for now. Doesn't seem that my "modules" idea is well supported by current dotnet templating engine

**Comment by isaacabraham** [comment]
@theimowski is this actually "in progress"? @colinbull any objections to us closing this?

**Comment by colinbull** [comment]
None from me. If I have ago I’ll re-open.
